packages/server-core/src/setting/client-setting/client-setting.seed.ts

```diff
@@ -69,6 +69,7 @@ export async function seed(knex: Knex): Promise<void> {
         themeSettings: JSON.stringify(defaultThemeSettings),
         themeModes: JSON.stringify(defaultThemeModes),
         key8thWall: process.env.VITE_8TH_WALL || '',
+        privacyPolicy: 'https://www.etherealengine.com/privacy',
```

Comment:
In seed file, we need to explicitly handle scenario where user data is already populated an we need to use insert/update privacyPolicy column. This will be the scenario for our dev/app/exp, etc. Else the column will remain empty.

Reply:
right. added this change!